soroban-env-host/src/e2e_invoke.rs

```diff
@@ -143,36 +140,31 @@ pub struct LedgerEntryLiveUntilChange {
 fn build_restored_key_set(
     budget: &Budget,
     resources: &SorobanResources,
-    resource_ext: &SorobanTransactionDataExt,
+    restored_rw_entry_ids: &[u32],
```

Comment:
These are indices for the read_write set right? Should this be called `restored_rw_entry_indices` like we already do in `InvokeHostFunctionRecordingModeResult`?